src/k8s/pkg/k8sd/api/cluster.go

```diff
@@ -45,6 +69,13 @@ func (e *Endpoints) getClusterStatus(s *state.State, r *http.Request) response.R
 				Type:    config.Datastore.GetType(),
 				Servers: config.Datastore.GetExternalServers(),
 			},
+			DNS:           featureStatuses["dns"],
```

Comment:
you can just retrieve the `statuses`, then do `statuses["dns"].ToAPI()`

Reply:
`ToAPI()` returns an `error`, we can remove it but I guess it's not a bad thing to have it there (for future)